google-cloud-logging/src/main/java/com/google/cloud/logging/Synchronicity.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.logging;
+
+public enum Synchronicity {
```

Comment:
I feel pretty strongly on this one: I have seen over-extended enumerations cause quite a bit of pain. In particular, when you end up adding values which specify an independent semantic dimension from the original dimension, you end up with a combinatorial cluster-f*. If we need new options, I think we should create a new enum & setting, unless the new option fits neatly within the "synchronicity" concept.